client/state/timezones/reducer.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+import { TIMEZONES_RECEIVE } from 'state/action-types';
+
+import {
+	rawOffsetsSchema,
+	labelsSchema,
+	continentsSchema
+} from './schema';
+
+export const rawOffsetReducer = createReducer( {}, {
+	[ TIMEZONES_RECEIVE ]: ( state, { rawOffsets = [] } ) => rawOffsets
+}, rawOffsetsSchema );
+
+export const labelsReducer = createReducer( {}, {
+	[ TIMEZONES_RECEIVE ]: ( state, { labels = {} } ) => labels
+}, labelsSchema );
+
+export const byContinentsReducer = createReducer( [], {
+	[ TIMEZONES_RECEIVE ]: ( state, { byContinents } ) => ( byContinents )
+}, continentsSchema );
```

Comment:
it should be fine @retrofox - did the linter complain about shadowing?

being explicit in the way you rewrote that is fine too